specification/network/resource-manager/Microsoft.Network/stable/2017-11-01/networkWatcher.json

```diff
@@ -1224,13 +1569,18 @@
       "description": "List of network watcher resources."
     },
     "TopologyParameters": {
-      "required": [
-        "targetResourceGroupName"
-      ],
       "properties": {
         "targetResourceGroupName": {
```

Comment:
[The example](https://github.com/chandrasekarsrinivasan/azure-rest-api-specs/blob/b4c0087e8eb00754cd83a7929e2d0a2e2586b82d/specification/network/resource-manager/Microsoft.Network/stable/2017-11-01/examples/NetworkWatcherTopologyGet.json#L9) has different field `TargetResourceGroup` instead of `TargetResourceGroupName`. Please fix the example.